documentation/book/proc-deploying-kafka-cluster-openshift.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-cluster.adoc
+
+[id='deploying-kafka-cluster-openshift-{context}']
+= Deploying the Kafka cluster to {OpenShiftName}
+
+{OpenShiftName} provides a template for deploying the Kafka cluster either in the {OpenShiftName} console or on the command-line.
+
+.Prerequisites
+
+* Before deploying a Kafka cluster, the Cluster Operator must be deployed.
+
+.Procedure
+
+. If you are planning to use the Kafka cluster for development or testing, create an ephemeral cluster
++
+[source,shell]
+----
+oc new-app strimzi-ephemeral
```

Comment:
This would create the cluster from the OpenShift templates. They have much less configuration options. And they need to be deployed into the catalog. I think this should be exactly the same as as the `proc-deploying-kafka-cluster-kubernetes.adoc` and just say `oc` instead of `kubectl. It should also refer to `xref:kafka_assembly_details[]`.